test/symmetries/fusiontrees.jl

```diff
@@ -223,13 +237,13 @@ using .TestSetup
             end
         end
     end
-    @testset "Fusion tree $Istr: elementary artin braid" begin
+    (UnitStyle(I) == SimpleUnit) && @testset "Fusion tree $Istr: elementary artin braid" begin
```

Comment:
This should really be considering the braiding style; it would also fail for something like `Vec[G]`. And since we now have `Vec[Z{N}]` in TensorKitSectors.jl, we should probably start testing things with this sector as well. Looking at the whole fusiontrees.jl test file, it is clear this file needs some massive reorganization that is too big to be fixed in this PR. 


```suggestion
    (BraidingStyle(I) isa HasBraiding) && @testset "Fusion tree $Istr: elementary artin braid" begin
```

Reply:
I planned on increasing code coverage of the TensorKit tests after QuantumKitHub/TensorKitSectors.jl#43 passes. I can then also look into adding other untested sectors (like the anyons in tensors tests).